atomic_reactor/plugins/generate_sbom.py

```diff
@@ -285,6 +289,37 @@ def get_unique_and_sorted_components(
 
         return unique_components
 
+    def push_sboms_to_registry(self):
+        docker_config = os.path.join(self.workflow.conf.registries_cfg_path, '.dockerconfigjson')
+
+        if not os.path.exists(docker_config):
+            self.log.warning("Dockerconfig json doesn't exist in : '%s'",  docker_config)
+            return
```

Comment:
is it okay not to fail here?

Reply:
well I'd say that pushing SBOM to registry is bonus,
so if configuration is using config.json format we push it to registry if not
I'd just skip it instead of failing as SBOM will be in brew anyway